net/vpnc/files/vpnc.sh

```diff
@@ -90,6 +91,11 @@ proto_vpnc_teardown() {
 
 	pwfile="/var/etc/vpnc-$config.conf"
 
+	json_get_var authfail authfail
+	if [ "${authfail:-0}" -gt 0 ]; then
+		proto_block_start "$config"
```

Comment:
It's `proto_block_restart`.  Apart from that, you need a way to know the interface was tear down because of auth failure  before doing this `authfail` config check.  `pppd` signifies this condition in its exit status (i.e. `11` and `19`).

This feature is good and reasonable, thanks, but _personally_ I would expect it being requested/implemented if it really bothers someone to keep the feature set compact and slim.  I myself came across such a situation using `pppd` against a local ISP :) (https://dev.openwrt.org/ticket/14144#comment:10)